### ISSUE TITLE
Adding info for torchscript exported model output

### ISSUE THREAD

**Issue by FrancescoMandru** [facebookresearch/detectron2#3988]
I run into your documentation about exporting and using detectron2 maskrcnn model in [production](https://detectron2.readthedocs.io/en/latest/tutorials/deployment.html). I wrote a simple code to export the model in `.pt` format as follows:

```python
torch_model = build_model(cfg)
DetectionCheckpointer(torch_model).resume_or_load(cfg.MODEL.WEIGHTS)
torch_model.eval()

# get a sample data
data_loader = build_detection_test_loader(cfg, cfg.DATASETS.TEST[0])
first_batch = next(iter(data_loader))

# trace both caffe2 and onnx
tracer = Caffe2Tracer(
    cfg, 
    torch_model, 
    first_batch
)

#caffe2_model = tracer.export_caffe2()
#onnx_model = tracer.export_onnx()
torchscript_model = tracer.export_torchscript()
torchscript_model.save("maskrcnn.pt")
```
Then, I loaded again the model as `loaded_model = torch.jit.load('maskrcnn.pt')` and tried to pass an Image in the format you provided:

```python
path_img = "my_image"
img = cv2.imread(path_img)
img = cv2.resize(img, (512,512), interpolation = cv2.INTER_AREA)
cuda0 = torch.device('cuda:0')
img = torch.as_tensor(img.astype("float32").transpose(2, 0, 1), device=cuda0)
img = torch.unsqueeze(img, axis=0)
im_info = torch.tensor(np.array((512,512,1.0)), device=cuda0, dtype=torch.float32)
im_info = torch.unsqueeze(im_info, axis=0)

out = loaded_model((img,im_info))
```

The output is a tuple which contain 4 tensors of shapes (100,4), (100), (100), (100, 3, 28, 28). As written [here](https://detectron2.readthedocs.io/en/latest/tutorials/deployment.html), 

> The converted models do not contain post-processing operations that transform raw layer outputs into formatted predictions. For example, the C++ examples only produce raw outputs (28x28 masks) from the final layers that are not post-processed, because in actual deployment, an application often needs its custom lightweight post-processing, so this step is left for users.

but I don't understand what I have to do in order to obtain the basic maskrcnn output at this point without any custom post-processing function, an example would be very useful. Thank you.
